### DISCUSSION TITLE
Move the POT Generator to the Tools menu

### DISCUSSION THREAD

**Discussion by timothyqiu** [godotengine/godot-proposals#10959] [Editor]
It is not only a hassle but also feels strange to go to the Localization tab of Project Settings dialog to generate/update the POT file.

I suggest moving this into Project > Tools menu.

For easier transition, we could also leave a button in the Localization tab as an alternative way to access that tool.

**Comment by dalexeev** [comment]
I think it would be convenient for a one-click[^1] update when you already have the POT generation parameters configured. But from a user experience perspective, we should consider the case when the user clicks this without any parameters configured. Also, as far as I remember, the target `*.pot` file is not currently remembered.

[^1]: Actually, it's 2 clicks and popup menu, but still faster than via Project Settings.

**Comment by shitake2333** [comment]
We only need to set the pot generation rules once, and then just click the `Generate` button, and the pot will be automatically generated and output to the specified location, without having to select where to output it every time like now.

We can also handle the generation of multiple pot files at the same time.

The editor settings section is used to configure the generation rules, and `Project > Tools menu` is used to generate and update pot files.
